### DISCUSSION TITLE
Is development dead on this? It's been over a year and a half since the last version

### DISCUSSION THREAD

**Discussion by cleverestx** [akhilrex/podgrab#296] [General]
I'm hoping not. This is a great little application. Is this not going to continue?

**Comment by Flashman00700** [comment]
Sadly, looks like it.

**Comment by Mindsgoneawol** [comment]
Crap. And I just found this awesome setup this morning. All it needs is a dlna server built in and it would be what I need! (atleast one that home assistant would see anyway.)

**Reply by jonasbg** [reply]
Im still using it daily, and have for the last two years. It will still work tomorrow as well ッ 
Integrate it with a better podcast listener app and you're all set ッ
